packages/core/admin/ee/server/services/review-workflows/workflows/index.js

```diff
@@ -19,8 +27,21 @@ const processFilters = ({ strapi }, filters = {}) => {
 
 module.exports = ({ strapi }) => {
   const workflowsContentTypes = workflowsContentTypesFactory({ strapi });
+  const limits = {
+    workflows: MAX_WORKFLOWS,
+    stagesPerWorkflow: MAX_STAGES_PER_WORKFLOW,
+  };
 
   return {
+    register({ workflows, stagesPerWorkflow }) {
+      if (!Object.isFrozen(limits)) {
+        limits.workflows = clampMaxWorkflows(workflows || limits.workflows);
```

Comment:
super minuscule thing, is it really necessary to have the clamp utility? We will only be usin git in the workflows/stages register and I feel it's more expressive to have 
`limits.workflows = clamp(1, MAX_WORKFLOWS, workflows ?? MAX_WORKFLOWS);`

Reply:
that or:
```js
    register({ workflows = MAX_WORKFLOWS, stagesPerWorkflow = MAX_STAGES_PER_WORKFLOW }) {
      if (!Object.isFrozen(limits)) {
        limits.workflows = clamp(1, MAX_WORKFLOWS, workflows);
```

Reply:
As we do not use Typescript, I would rather prefer to use `||` to avoid any `NaN` because we pass a null value. 😅 

The clamp function is important as it will ensure that the value is between 1 and MAX_WORKFLOWS.